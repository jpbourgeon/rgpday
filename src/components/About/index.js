import React from 'react'
import PropTypes from 'prop-types'
import Link from '@material-ui/core/Link'
import { withStyles } from '@material-ui/core/styles'
import Typography from '@material-ui/core/Typography'
import Paper from '@material-ui/core/Paper'
import Grid from '@material-ui/core/Grid'
import Divider from '@material-ui/core/Divider'
import Avatar from '@material-ui/core/Avatar'
import Info from '@material-ui/icons/InfoOutlined'
import CallToAction from '../CallToAction'
import JPB from './images/avatar.jpg'

const styles = theme => ({
  layout: {
    width: 'auto',
    marginLeft: theme.spacing.unit * 3,
    marginRight: theme.spacing.unit * 3,
    [theme.breakpoints.up(1100 + theme.spacing.unit * 3 * 2)]: {
      width: 1100,
      marginLeft: 'auto',
      marginRight: 'auto'
    }
  },
  paper: {
    backgroundSize: `100% 100%`,
    marginBottom: theme.spacing.unit * 4,
    padding: theme.spacing.unit * 4,
    textAlign: 'justify'
  },
  title: {
    marginBottom: theme.spacing.unit * 3,
    order: 0
  },
  titleIcon: {
    fontSize: '2.5rem',
    marginBottom: '-0.5rem'
  },
  avatar: {
    order: 1,
    minHeight: `${110 + (theme.spacing.unit * 6)}px`,
    [theme.breakpoints.down('sm')]: {
      marginBottom: `-${theme.spacing.unit * 4}px`
    }
  },
  avatarIcon: {
    width: 110,
    height: 110,
    margin: 'auto'
  },
  post: {
    paddingBottom: theme.spacing.unit * 6,
    order: 2,
    [theme.breakpoints.down('sm')]: {
      order: 3
    }
  },
  cta: {
    order: 3,
    [theme.breakpoints.down('sm')]: {
      order: 2,
      marginBottom: `-${theme.spacing.unit * 4}px`
    }
  }
})

const About = props => {
  const { classes } = props
  return (
    <div className={classes.layout}>
      <main>
        <Paper className={classes.paper}>
          <Grid container className={classes.container}>
            <Grid item xs={12} className={classes.title}>
              <Typography variant='h4' gutterBottom>
                <Info className={classes.titleIcon} />&nbsp;À propos
              </Typography>
              <Divider />
            </Grid>
            <Grid item xs={12} sm={4} md={2} className={classes.avatar}>
              <Avatar alt='Jean-Philippe Bourgeon' src={JPB} className={classes.avatarIcon} />
            </Grid>
            <Grid item xs={12} sm={12} md={6} className={classes.post}>
              <Typography variant='body1' gutterBottom>
                En qualité de <strong>Responsable du pôle Systèmes d'information</strong>, gestion des données et des méthodes au Centre de gestion de la fonction publique territoriale de Saône-et-Loire, <strong>je développe avec mon équipe</strong>, des solutions pour accompagner les collectivités territoriales du département dans la gestion quotidienne de leurs données. Bien sûr, un des premiers services que nous rendons est <strong>la mutualisation du Délégué à la Protection des Données</strong> : nous accompagnons nos adhérents dans leur mise en conformité avec le RGPD.
              </Typography>
              <Typography variant='body1' gutterBottom>
                Cette journée a été conçue pour répondre à un problème <strong>simple et concret</strong>. Une fois renseigné et formé aux concepts de la protection des données à caractère personnel, <strong>par où doit-on commencer la mise en conformité</strong> ? Comment cette démarche est-elle liée aux autres outils de gestion de l'information dans l'entreprise ?
              </Typography>
              <Typography variant='body1' gutterBottom>
                Le <strong>RGPDay</strong> s'adresse à <strong>tous les organismes de formation</strong>, initiale ou continue, dans le domaine de la gestion de l'information, qui souhaitent proposer <strong>une expérience du RGPD pratique et ludique</strong> à leurs étudiants, dans le prolongement de leur formation théorique.
              </Typography>
              <Typography variant='body1' gutterBottom>
                --<br />
                Jean-Philippe Bourgeon<br />
                <Link href='https://www.linkedin.com/in/jpbourgeon'>Profil LinkedIn</Link>
              </Typography>
            </Grid>
            <Grid item xs={12} sm={8} md={4} className={classes.cta}>
              <CallToAction xs={12} sm={10} md={10} />
            </Grid>
          </Grid>
        </Paper>
      </main>
    </div>
  )
}

About.propTypes = {
  classes: PropTypes.object.isRequired
}

export default withStyles(styles)(About)
