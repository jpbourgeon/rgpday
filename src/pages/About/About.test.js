/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import About from './'
import jssSerializer from '../../utils/jssSnapshotSerializer'

expect.addSnapshotSerializer(jssSerializer)

describe('The About component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<About />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
