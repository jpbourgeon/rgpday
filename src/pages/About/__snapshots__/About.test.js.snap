// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The About component should render correctly 1`] = `
<div
  className="layout-1"
>
  <main>
    <div
      className="root-9 elevation2-13 rounded-10 paper-2"
    >
      <div
        className="container-36"
      >
        <div
          className="item-37 12-76 title-3"
        >
          <h4
            className="root-133 h4-148 gutterBottom-160"
          >
            <svg
              aria-hidden="true"
              className="root-169 titleIcon-4"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0V0z"
                fill="none"
              />
              <path
                d="M11 7h2v2h-2zM11 11h2v6h-2z"
              />
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
              />
            </svg>
             À propos
          </h4>
          <hr
            className="root-178"
          />
        </div>
        <div
          className="item-37 12-76 4-82 2-94 avatar-5"
        >
          <div
            className="root-183 avatarIcon-6"
          >
            <img
              alt="Jean-Philippe Bourgeon"
              className="img-185"
              src="avatar.jpg"
            />
          </div>
        </div>
        <div
          className="item-37 12-76 12-90 6-98 post-7"
        >
          <p
            className="root-133 body1-142 gutterBottom-160"
          >
            En qualité de 
            <strong>
              Responsable du pôle Systèmes d'information
            </strong>
            , gestion des données et des méthodes au Centre de gestion de la fonction publique territoriale de Saône-et-Loire, 
            <strong>
              je développe avec mon équipe
            </strong>
            , des solutions pour accompagner les collectivités territoriales du département dans la gestion quotidienne de leurs données. Bien sûr, un des premiers services que nous rendons est 
            <strong>
              la mutualisation du Délégué à la Protection des Données
            </strong>
             : nous accompagnons nos adhérents dans leur mise en conformité avec le RGPD.
          </p>
          <p
            className="root-133 body1-142 gutterBottom-160"
          >
            Cette journée a été conçue pour répondre à un problème 
            <strong>
              simple et concret
            </strong>
            . Une fois renseigné et formé aux concepts de la protection des données à caractère personnel, 
            <strong>
              par où doit-on commencer la mise en conformité
            </strong>
             ? Comment cette démarche est-elle liée aux autres outils de gestion de l'information dans l'entreprise ?
          </p>
          <p
            className="root-133 body1-142 gutterBottom-160"
          >
            Le 
            <strong>
              RGPDay
            </strong>
             s'adresse à 
            <strong>
              tous les organismes de formation
            </strong>
            , initiale ou continue, dans le domaine de la gestion de l'information, qui souhaitent proposer 
            <strong>
              une expérience du RGPD pratique et ludique
            </strong>
             à leurs étudiants, dans le prolongement de leur formation théorique.
          </p>
          <p
            className="root-133 body1-142 gutterBottom-160"
          >
            --
            <br />
            Jean-Philippe Bourgeon
            <br />
            <a
              className="root-133 colorPrimary-163 inline-168 root-186 underlineHover-188"
              href="https://www.linkedin.com/in/jpbourgeon"
            >
              Profil LinkedIn
            </a>
          </p>
        </div>
        <div
          className="item-37 12-76 8-86 4-96 cta-8"
        >
          <div
            className="container-36 40-62 column-39 center-44"
          >
            <div
              className="item-37 12-76 10-88 10-102"
            >
              <div
                className="root-9 elevation1-12 rounded-10 cta-191"
              >
                <h6
                  className="root-133 h6-150"
                >
                  <a
                    className="root-133 colorSecondary-164 inline-168 root-186 underlineHover-188"
                    href="/contact"
                    onClick={[Function]}
                  >
                    Contactez-moi !
                  </a>
                  <br />
                  pour organiser le RGPDay dans votre établissement
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
